homeworkW13D5/src/homeworkW13D5/ORM/Person.java

```diff
@@ -0,0 +1,37 @@
+package homeworkW13D5.ORM;
+
+public class Person extends Model {
+
+	private String tableName;
+	private String value;
+	private Integer id;
+
+	public Person(String tableName, String value, Integer id) {
```

Comment:
Constructor should not contain name of the table, it should be known in advanced and `hard-coded` in method's constructor.